spec/motion-spec/mac_bacon_spec.rb

```diff
@@ -162,7 +162,7 @@ def triggerChange
           end
         end
 
-        behaves_like 'waiting in before/after filters'
+        include_examples 'waiting in before/after filters'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.